test/testutils.cpp

```diff
@@ -334,56 +334,11 @@ class TestUtils : public TestFixture {
         }
     }
 
-    void ptrToString() const
+    void id_string() const
```

Comment:
The test should preserve the actual tested cases (min, max, 32-bit/64-bit, nullptr, etc.).

Reply:
sure